package.json

```diff
@@ -30,8 +30,7 @@
     "left-pad": "^1.1.3",
     "semver": "^5.3.0",
     "solidity-sha3": "^0.4.1",
-    "truffle-artifactor": "^2.1.2",
```

Comment:
Just checkin' in here: is the artifactor not used by the pkg?

Reply:
If it is, I cannot see where it is used.